swatch-contracts/src/main/java/com/redhat/swatch/contract/repository/SubscriptionRepository.java

```diff
@@ -90,7 +90,12 @@ public List<BillingAccountInfoDTO> findBillingAccountInfo(
       String orgId, Optional<String> productTag) {
     StringBuilder query =
         new StringBuilder(
-            "select distinct subscription.orgId, subscription.billingAccountId, subscription.billingProvider, productTag from SubscriptionEntity subscription join subscription.offering offering join offering.productTags productTag where orgId = ?1");
+            "select distinct subscription.orgId, subscription.billingAccountId, subscription.billingProvider, productTag from SubscriptionEntity subscription "
+                + "join subscription.offering offering "
+                + "join offering.productTags productTag "
+                + "where orgId = ?1 "
+                + "and subscription.startDate <= CURRENT_TIMESTAMP "
+                + "and subscription.endDate >= CURRENT_TIMESTAMP");
```

Comment:
Empty end dates need to be taken into consideration for this.  See the comment with the test plan/expected results.

If a subscription started on May 7, 2025 and doesn't have an end date, I would expect its billing account id to be part of the results.

Reply:
Done